src/amo/components/AutoSearchInput/index.js

```diff
@@ -96,6 +98,7 @@ export class AutoSearchInputBase extends React.Component<Props, State> {
 
   static defaultProps = {
     debounce: defaultDebounce,
+    inputLabelIsHidden: false,
```

Comment:
Specifying `inputLabelIsHidden={true}` (or, really, just `<AutoSearchInput inputLabelIsHidden {...otherProps} />` is a weird API to me. It might be better to have this be `showInputLabel` so it would be `<AutoSearchInput showInputLabel={false} {...otherProps} />` when used. I think setting something to `false` for it to be hidden makes a bit more sense cognitively.

Or at least have it be more instructive if it's a boolean you set to true... maybe `hideInputLabel` or something.